homeassistant/components/climate/mill.py

```diff
@@ -114,13 +114,16 @@ def device_state_attributes(self):
         """Return the state attributes."""
         if self._heater.room:
             room = self._heater.room.name
+            avg_room_temp = self._heater.room.avg_temp
         else:
             room = "Independent device"
+            avg_room_temp = None
         return {
             "room": room,
             "open_window": self._heater.open_window,
             "heating": self._heater.is_heating,
             "controlled_by_tibber": self._heater.tibber_control,
+            "avg_room_temp": avg_room_temp,
```

Comment:
We usually don't populate device state attributes with items where the value is `None`. Maybe we should check that here?

Reply:
Ok, so if it is None, we do not add it?

Reply:
Yeah, that's an existing pattern.